### ISSUE TITLE
ZMQ and other data-science-y topics

### ISSUE THREAD

**Issue by kytrinyx** [exercism/discussions#21]
I got an email today that quoted a discussion that someone had:

> I’m playing a bit with http://exercism.io which is quite nice and
> wonder if it would make sense to come up with some exercises on
> ZeroMQ? Probably needs some talking to the makers since exercism.io
> as-is is about languages, see http://exercism.io/languages, but maybe
> they are open to adding new sections like on ZeroMQ with separate
> languages inside. More sections could make sense (the number of
> languages is limited), and I’d also like to see something for more
> data science related content. Any ideas?

My immediate thoughts are:

- this would be fun and interesting and useful
- this would not work out of the box and would require some work to make happen
- this might make it harder to explain what exercism is
- this would make a fantastic stand-alone project

**Comment by ryanplusplus** [comment]
Sounds like there are a lot of ways in which this doesn't really fit the story and architecture of exercism. I think it's a cool idea, but I agree that it should be an independent project leveraging some of the exercism code.

**Comment by kytrinyx** [comment]
I'm not even sure any of the existing code would help--it's such a chaotic prototype right now, with a ton of scar tissue and left-overs from experiments. My guess is that exercism could be an inspiration, but it would be easier to design/write it from scratch.

**Comment by IanWhitney** [comment]
Yeah, having worked with (and liked) ZMQ, I have no idea how we'd do anything with it in Exercism. It'd be like implementing problems in HTTP.
